### ISSUE TITLE
Include UC Browser 

### ISSUE THREAD

**Issue by awebdeveloper** [jakearchibald/isserviceworkerready#90]
Plz include info for uc browser as it's the biggest browser in India, china and this part of the world

**Comment by hax** [comment]
You could check https://ispwaready.toxicjohann.com/ 
This site list the status of ServiceWorker and other PWA related tech of most mainstream browsers in China.
I expect the data of that [project](https://github.com/toxic-johann/is-pwa-ready) could be merged to this project in the future.

**Comment by toxic-johann** [comment]
Data in [isserviceworkerready](https://github.com/jakearchibald/isserviceworkerready) is the subset of [is-pwa-ready](https://github.com/toxic-johann/is-pwa-ready). So it's totally possible to merge. I may launch a  pull request later. But it seems that Jake has not run this project for a long time. So @awebdeveloper you can just go to [https://ispwaready.toxicjohann.com](https://ispwaready.toxicjohann.com)

**Comment by awebdeveloper** [comment]
@toxic-johann  plz include a title attr so that we know what browser its is. Some of these logos are alien to us.

**Comment by toxic-johann** [comment]
@awebdeveloper Thanks for ur advice. I will update it tonight~

**Comment by awebdeveloper** [comment]
@toxic-johann  also mention what each color are yellow, green, grey etc

**Comment by toxic-johann** [comment]
@awebdeveloper Yes. I agree. [https://github.com/toxic-johann/is-pwa-ready/issues/2](https://github.com/toxic-johann/is-pwa-ready/issues/2). If you find some more shortage. Just wrote add that issue.~~

**Comment by toxic-johann** [comment]
@awebdeveloper I have updated is-pwa-ready~ I think now it's better~

**Comment by jakearchibald** [comment]
Hi, I'm in the middle of a huge I/O deadline, so please don't assume this project is dead because I don't reply within 3 days.

Also, if you want to discuss other projects, please do it in the repo for that project, especially since the other project claims to be some sort of competitor.

**Comment by jakearchibald** [comment]
If you want to speed up action on this issue, PRs are helpful.

**Comment by awebdeveloper** [comment]
@toxic-johann now it makes more sense to merge the two.

Also @jakearchibald would be open to merging /adding more browsers

**Comment by toxic-johann** [comment]
@jakearchibald it's not kind of competitor. I will take a pull request this weekend.

**Comment by toxic-johann** [comment]
@jakearchibald @awebdeveloper I have just take a pull request. And the check passed. If have some problem, just contact me. But as UC is still under development, the data may change soon. What's more, it may be a different version between countries on some API,  such as push API. I will update at that time.
